Twig/Extension/CmfSeoExtension.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Ben Glassman <bglassman@gmail.com>
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Twig\Extension;
+
+use Symfony\Cmf\Bundle\SeoBundle\SeoPresentationInterface;
+
+class CmfSeoExtension extends \Twig_Extension
+{
+    /**
+     * @var SeoPresentationInterface
+     */
+    protected $seoPresentation;
+
+    /**
+     * @param SeoPresentationInterface $seoPresentation
+     */
+    public function __construct(SeoPresentationInterface $seoPresentation)
+    {
+        $this->seoPresentation = $seoPresentation;
+    }
+
+    /**
+     * @{inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('cmf_seo_update_seo_page',
```

Comment:
I agree one the name, it seems to be a case where the fact that sonata seo bundle is being used is being exposed perhaps unnecessarily. I think update rather than set may be more correct, however, because some of the data is merged rather than overriden? I'm not 100% sure on this. I think you are saying this but if we are going to change the name here we should probably also change the SeoPresentationInterface method as well and deprecate the old one.